### DISCUSSION TITLE
Unable to unblock domain

### DISCUSSION THREAD

**Discussion by agneevX** [AdguardTeam/AdGuardHome#3920] [Q&A]
Hello,

I want to unblock a specific domain, but I still receive a `NXDOMAIN` response

```
# Block DNS over HTTPS domains
||doh.*^$dnsrewrite=NXDOMAIN;;

# Unblock OpenDNS
@@||doh.opendns.com^$important
```

**Comment by ainar-g** [comment]
`$dnsrewrite` rules have a higher priority and their own syntax for disabling rules, since they are special.  In your case, something like this should work:

```none
||doh.*^$dnsrewrite=NXDOMAIN;;
@@||doh.opendns.com^$dnsrewrite
```
